website:fix validation failures

jgil-r opened the pull request:
In install option links, and feature comparison
table I fixed aria-labelled-by attribute to correct spelling
aria-labelledby. Fixed time element to have correct datetime attribute,
which translates dates into machine-readable format. Lastly, changed
obsolete tt element, in favor of code element. #335

jgil-r commented:
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

strager commented on website/public/index.ejs.html:
```diff
@@ -150,7 +153,7 @@ <h2>Install quick-lint-js</h2>
               src="gnome-terminal.svg"
               alt=""
-              aria-labeled-by="install-cli"
+              aria-labelledby="install-cli"
```
🤦‍♀️

strager reviewed (approved):
Looks good! Shall I merge it now?

strager commented:
> Removed invalid aria-labelledby attribute from img tags in install
options links, and added alt attribute instead.

This description doesn't match the code.

jgil-r commented:
> > Removed invalid aria-labelledby attribute from img tags in install
> > options links, and added alt attribute instead.
> 
> This description doesn't match the code.

Yeah I just fixed it actually. I forgot to rebuild the site before the latest commit though, is that why there is one failing check?

strager commented:
> I forgot to rebuild the site before the latest commit though, is that why there is one failing check?

No. The CentOS check failure is unrelated to your PR. It's happening on master. (I just noticed it.)

jgil-r commented:
Okay, it should be good to go then. Would you like me to edit the main comment with a description of the new changes from most recent commit?

strager commented:
> Okay, it should be good to go then. Would you like me to edit the main comment with a description of the new changes from most recent commit?

Yes. This message will serve as the published commit's message.

jgil-r commented:
Done, should be good now.
